.travis.yml

```diff
@@ -0,0 +1,37 @@
+# After changing this file, check it on:
+#   http://lint.travis-ci.org/
+language: python
+
+# Run jobs on container-based infrastructure, can be overridden per job
+sudo: false
+
+matrix:
+  include:
+    - python: 2.7
```

Comment:
Can you get rid of python 2? It is old and we won't be supporting it. We should test python 3.4, 3.5, 3.6 IMO